### PR TITLE
NF: simplify Time and improve lint message

### PR DESCRIPTION
**Pull request by Arthur-Milchior**
Lint message around system time was not update when Time was moved to Collection
from TimeManager.

In my opinion, code is more readable if you just uses `Time.s` and `Time.ms`
than `TimeManager.Time.intTime`. On top of that, it becomes clear when time is
returned in seconds.

This way, most developer can just write code without needing to have any
knowledge about time management, unless they have a test that requires to
manipulate it explicitly.

As a next step, we may try to figure out whether methods that require Time
really need them. Most seems to directly gets `TimeManager.time` in all
call. But this seems less important than lint (this one is mandatory) and
simplifying our abstraction for new contributors.

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/anki/CardBrowser.kt [hunk 1]
```diff
@@ -2051,9 +2051,9 @@ open class CardBrowser :
                     showProgressBar()
                 }
                 // Also start rendering the items on the screen every 300ms while scrolling
-                val currentTime = SystemClock.elapsedRealtime()
-                if (currentTime - mLastRenderStart > 300 || lastVisibleItem + 1 >= totalItemCount) {
-                    mLastRenderStart = currentTime
+                val Time = SystemClock.elapsedRealtime()
+                if (Time - mLastRenderStart > 300 || lastVisibleItem + 1 >= totalItemCount) {
+                    mLastRenderStart = Time
```
Comment (divyansh-dxn) [inline AnkiDroid/src/main/java/com/ichi2/anki/CardBrowser.kt:2056]: @Arthur-Milchior have you kept variable name with first letter caps for any specific reason?

Comment (Arthur-Milchior) [reply AnkiDroid/src/main/java/com/ichi2/anki/CardBrowser.kt:2056]: Oops, no. I wanted to rename currentTime method, I didn't saw there was variables too

**Comment by Arthur-Milchior** [comment]
This one was strange. Backticks in Description made the test failed. For some reason, Kotlin would just not print two backticks. However if the string has a single backtick, it works. No idea why. Not important for this case either

**Comment by ShridharGoel** [comment]
Conflicts need to be resolved.

**Comment by Arthur-Milchior** [comment]
Done

File: AnkiDroid/src/main/java/com/ichi2/libanki/stats/AdvancedStatistics.java [hunk 2]
```diff
@@ -1067,9 +1067,9 @@ public int getInitialFactor() {
         public int getToday(long collectionCreatedTime) {
             Timber.d("Collection creation timestamp: %d", collectionCreatedTime);
 
-            long currentTime = TimeManager.INSTANCE.getTime().intTime();
-            Timber.d("Now: %d", currentTime);
-            return (int) ((currentTime - collectionCreatedTime) / SECONDS_PER_DAY);
+            long CurrentTime = TimeManager.INSTANCE.getTime().intTime();
```
Comment (lukstbit) [inline AnkiDroid/src/main/java/com/ichi2/libanki/stats/AdvancedStatistics.java:1070]: Why the change to uppercase and why not replace `TimeManager.INSTANCE.getTime().intTime()` with the new method?

**Review by lukstbit — CHANGES_REQUESTED** [review]

**Comment by divyansh-dxn** [comment]
Nothing more from me, just the question lukstbit asked.

File: AnkiDroid/src/main/java/com/ichi2/libanki/stats/AdvancedStatistics.java (hunk 2 above)
Comment (Arthur-Milchior) [reply AnkiDroid/src/main/java/com/ichi2/libanki/stats/AdvancedStatistics.java]: No idea for the cap.
I would assume the absence of correction was that I mostly automated it for kotlin, but I may be wrong.

**Review by david-allison — CHANGES_REQUESTED** [review]
These should be functions, not properties.

> Prefer a property over a function when the underlying algorithm:
> * does not throw
> * is cheap to calculate (or cached on the first run)
> * **returns the same result over invocations if the object state hasn't changed**

https://kotlinlang.org/docs/coding-conventions.html#functions-vs-properties

Emphasis mine

----
non-blocking, for discussion

* We're moving away from Anki desktop's naming
* We should `typealias` the return values

**Comment by Arthur-Milchior** [comment]
What kind of typealias do you want? milisecond and Second?

**Comment by david-allison** [comment]
`EpochS` and `EpochMs` seems sensible

**Pull request closed without merging**